src/components/settings/SettingsPage.js

```diff
@@ -58,6 +64,13 @@ const SettingsPage = ({ desktop }) => {
   const [overrideDataLanguage, setOverrideDataLanguage] = useState(false)
   const { t, i18n } = useTranslation()
 
+  const updateUnitsSetting = (index) => {
+    for (const [key, value] of Object.entries(METRICS_OPTIONS)) {
+      if (index === value) {
+        dispatch(updateSettings({ ['distanceUnit']: key }))
```

Comment:
You actually don't need to put distanceUnit in quotes here. That's called a computed property name: https://ui.dev/computed-property-names/

It's already a string so you can just use it as a literal.

Reply:
I tried the literal, but was thrown an error so will keep it as is.